config/config.exs

```diff
@@ -1,17 +1,35 @@
 import Config
 
+config :wanda, Wanda.Messaging.ProcessCache,
+  # When using :shards as backend
+  # backend: :shards,
+  # GC interval for pushing new generation: 12 hrs
+  gc_interval: :timer.hours(12),
+  # Max 1 million entries in cache
+  max_size: 1_000_000,
+  # Max 2 GB of memory
+  allocated_memory: 2_000_000_000,
+  # GC min timeout: 10 sec
+  gc_cleanup_min_timeout: :timer.seconds(10),
+  # GC max timeout: 10 min
+  gc_cleanup_max_timeout: :timer.minutes(10)
+
 config :wanda, Wanda.Messaging.Publisher, adapter: Wanda.Messaging.Adapters.AMQP
 
 config :wanda,
-  children: [Wanda.Messaging.Adapters.AMQP.Publisher, Wanda.Messaging.Adapters.AMQP.Consumer]
+  children: [
+    Wanda.Messaging.Adapters.AMQP.Publisher,
+    Wanda.Messaging.Adapters.AMQP.Consumer,
+    Wanda.Messaging.ProcessCache
+  ]
 
 config :wanda, :messaging,
   adapter: Wanda.Messaging.Adapters.AMQP,
   amqp: [
     consumer: [
       queue: "events_wanda",
       exchange: "events",
-      routing_key: "checks.*",
+      routing_key: "checks.executions",
```

Comment:
No strong opinions about the routing key.

It can be different things depending on what we want to achieve.
`checks.*` would match 
`checks.execution_requested`, `checks.facts_gathered` 

`checks.#` would match 
`checks.execution_requested.<execution_id>`, `checks.facts_gathered.<agent_id>`

sample [here](https://www.rabbitmq.com/tutorials/tutorial-five-elixir.html)

In this example a fixed `checks.executions` is just to try it out.
The fact is that wanda should be able to consume messages from the web and the agents
- web -> `ExecutionRequested`
- agents -> `FactsGathered`